azure-arc-for-servers/get-kvsecrets-from-arc-servers.ps1

```diff
@@ -4,62 +4,88 @@ function Get-AccessToken {
     param (
         [Parameter(Mandatory=$true)]
         [string] $audience,
-        [string] $apiVersion = "2019-08-15"
+        [string] $apiVersion = '2019-08-15'
     )
 
     $azcmagent = azcmagent show
```

Comment:
maybe check for azcmagent is available, if not error out.

```powershell
$agentExePath = Get-Command -Type Application -Name 'azcmagent'

if (-not $agentExePath)
{
 throw 'azcmagent not avaliable'
}
```

Reply:
excellent feedback. thx!